ReactiveCocoa/Swift/EventLogger.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  EventLogger.swift
+//  ReactiveCocoa
+//
+//  Created by Rui Peres on 30/04/2016.
+//  Copyright © 2016 GitHub. All rights reserved.
+//
+
+import Foundation
+
+/// Used with the `debug` operator to customize
+/// the `Event` string.
+///
+/// This is useful, when you want to do more than
+/// just printing to the standard output.
+public protocol EventLogger {
```

Comment:
I am ok with going with the function, but I am not completely sold. I find it more idiomatic, to with a protocol approach on this.